lte/gateway/python/magma/pipelined/tests/ng_node_rpc_servicer.py

```diff
@@ -0,0 +1,132 @@
+"""
```

Comment:
Should this be named test_ng_node_rpc_servicer.py

Reply:
I tried keeping the name as test_node_rpc_servicer.py. Following were the observations :
1. When tried to run the testcase individually the testcase is passing fine.
2. But when launched under "python test_all" the testcase is hanging in "self._rpc_server.start()". So applied the similar approach as used in "mobilityd/tests/rpc_servicer_tests.py".

test_all executes all cases using files with test_* files. With this name this test is excluded.

Reply:
Yeah in other pipelined tests we use start_pipelined.sh to not initiate the default pipelined rpc server. Maybe we should add something like this but I'm okay with this for now